reg/drone/service/gnss/Heartbeat.0.1.uavcan

```diff
@@ -0,0 +1,16 @@
+# Optional, extended information on the performance of the GNSS sensor.
+
+reg.drone.service.common.Heartbeat.0.1 heartbeat
+
+Sources.0.1 sources
+
+DilutionOfPrecision.0.1 dop
+
+FixStatus.0.1 fix_status
+
+uint8 num_visible_satellites  # The number of space vehicles visible when the solution was computed.
+uint8 num_used_satellites     # The number of space vehicles whose signals have been utilized to derive the solution.
+
+bool rtk_fix    # True if the current mode is RTK and an RTK fix has been established. Also see Sources.
```

Comment:
I am curious to know why `rtk_fix` is singled out here and not part of `FixStatus.0.1`?

Reply:
The fix status is intended to be an enumeration so I moved it into a dedicated type. As I just wrote above, the current proposal might be over-engineered slightly because the FixStatus can be replaced with a boolean flag next to `rtk_fix` without reducing the utility of the GNSS service. Unless you object, I would add `bool fix` next to this flag instead.

Reply:
Sounds good to me 👍 I'm still wondering why `rtk_fix` requires it's own flag in addition to `fix`? If `fix` is set and `rtk_base` is set, wouldn't that be the same as setting `fix,` `rtk_fix`, and `rtk_base`?

Reply:
- `fix` indicates that a standalone navigation solution is available.
- `rtk_fix` indicates that an RTK fix is established and the integer wavelength ambiguity is resolved.
- `sources.rtk_base` indicates that base station data (usually encapsulated into an RTCM stream) is available to the receiver, but it says nothing about the status of the navigation solution.

Reply:
Thank you for the clarification :+1: